### ISSUE TITLE
NVDA is not announcing the 'Grant access' dialog.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#17167]
### Steps to reproduce:
Turn on NVDA.

1. Open PowerPoint Application in windows.
2. Navigate to the comment option using the Tab key and mention someone who does not have access to the document.
3. Navigate to the Post button using tab key and activate it using Enter key.
4. 'Grant access' dialog will be appeared.
5. Observe whether NVDA is reading the information of 'Grant access' dialog or not.
### Actual behavior:
NVDA is not announcing the 'Grant access' dialog. When the dialog opens and on pressing tab key, NVDA remains silent.

### Expected behavior:
NVDA should announce the 'Grant access' dialog information. When pressing tab key, NVDA Should announce 'Share and Notify' button information.

### NVDA logs, crash dumps and other attachments:
<img width="960" alt="NVDA is not announcing the information in the 'Share' dialog popup  (1)" src="https://github.com/user-attachments/assets/e925026a-4438-43b3-b1eb-1d46cfd228a6">

https://github.com/user-attachments/assets/8f93ce12-a5b3-4511-bcd9-b308406778a0

**Comment by XLTechie** [comment]
Does Narrator read this dialog?

**Comment by msftedad** [comment]
@XLTechie, Yes Narrator is announcing the information after the dialog appears.
